### PR TITLE
Added link to Turkish translation.

### PR DESCRIPTION
**Pull request by LeaveNhA**
I've just started the translations.
I think showing the in-progress translation might increase the chances to get attention and help.

BTW: Thank you for this magical document. It was my second true-gide to get into professional Clojure experience.

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -139,6 +139,7 @@ Translations of the guide are available in the following languages:
 * https://github.com/theSkilled/clojure-style-guide/blob/pt-BR/README.md[Portuguese] (Under progress)
 * https://github.com/Nondv/clojure-style-guide/blob/master/ru/README.md[Russian]
 * https://github.com/jeko2000/clojure-style-guide/blob/master/README.md[Spanish]
+* https://github.com/LeaveNhA/clojure-style-guide/blob/master/README.md[Turkish]
```
Comment (bbatsov) [inline README.adoc:142]: Probably you meant to put `README.adoc` here? The other links are outdated, as they are from the time before the conversion to AsciiDoc.

Comment (LeaveNhA) [reply README.adoc:142]: I thought that there is an unwritten rule or some conversion to create translation/content.
I can change my repository to serve an ASCIIDoc file. Thank you for pointing out.

**Comment by bbatsov** [comment]
Thanks!

**Comment by LeaveNhA** [comment]
Not yet thank me, I want to translate and localised the content and find someone to keep it up-to-date.

Thank you, for creating such a guide!

**Pull request merged**